.jenkins/pytorch/build.sh

```diff
@@ -43,22 +43,24 @@ cmake --version
 pip install -r requirements.txt || true
 
 if [[ "$BUILD_ENVIRONMENT" == *rocm* ]]; then
+
+  if [[ "$BUILD_ENVIRONMENT" == *centos* ]]; then
```

Comment:
Since we are going to build new docker images with this PR, let's export these in the docker file.

Reply:
Which file is that?

Reply:
the new docker/caffe2/jenkins/centos-rocm/Dockerfile (and also the ubuntu-rocm/Dockerfile

Reply:
They should be exported already in both of them, see lines 45/46 in `docker/caffe2/jenkins/centos-rocm/Dockerfile` and lines 10/11 in `docker/caffe2/jenkins/ubuntu-rocm/Dockerfile `

Reply:
sorry I meant LC_ALL and LANG environment variables, not BUILD_ENVIRONMENT